KinEcosystem/ViewModel/OrderViewModel.swift

```diff
@@ -50,16 +58,28 @@ class OrderViewModel {
             details = ""
         }
         color = indicatorColor
+        
         title = model.title.attributed(18.0, weight: .regular, color: titleColor) +
                 details.attributed(14.0, weight: .regular, color: detailsColor)
         var subtitleString = model.description_
         if let shortDate = Iso8601DateFormatter.shortString(from: model.completion_date as Date) {
             subtitleString = subtitleString + " - " + shortDate
         }
+        
         subtitle = subtitleString.attributed(14.0, weight: .regular, color: .kinBlueGreyTwo)
         
-        amount = ((model.offerType == .earn ? "+" : "-") + "\(Decimal(model.amount).currencyString()) ").attributed(16.0, weight: .medium, color: .kinBlueGreyTwo)
-
+        var amountOperator = ""
+        switch model.offerType {
```

Comment:
also here, will we have an amount in failure case?

Reply:
Let's talk about it when you come back